### PR TITLE
#1190 Check if a new CLI version is available once a day

### PR DESCRIPTION
**Pull request by agrimmer**

### CODE REVIEW

File: cli/utils/version/version_checker.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	if err != nil {
+		return availableNewestVersions{}, fmt.Errorf("error when analyzing the available versions: %v", err)
+	}
+	return res, nil
+}
+
+const newCompatibleVersionMsg = `keptn version %s is available! Please visit https://keptn.sh for more information.`
+const newIncompatibleVersionMsg = `keptn version %s is available! Please note that this version is incompatible with your Keptn cluster` +
+	`version and requires to update the cluster too. Please visit https://keptn.sh for more information.`
+const disableMsg = `To disable this notice, run: 'keptn set config AutomaticVersionCheck false'`
```
Comment (christian-kreuzberger-dtx) [inline cli/utils/version/version_checker.go]: Please make the calling binary `keptn` a variable.

E.g., someone downloads the keptn cli and renames it into `keptn06` and even calling it from the local directory, then the text should be 
``To disable this notice, run: './keptn06 set config AutomaticVersionCheck false'``

**Comment by christian-kreuzberger-dtx** [comment]
I think this part still needs to be implemented:
```
$> keptn version
CLI version: 0.5.1 (Update available: 0.5.2)
```

File: cli/utils/version/version_checker.go [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+	res, err := getAvailableVersions(usedVersionString, *cliVersionInfo)
+	if err != nil {
+		return availableNewestVersions{}, fmt.Errorf("error when analyzing the available versions: %v", err)
+	}
+	return res, nil
+}
+
+const newCompatibleVersionMsg = `keptn version %s is available! Please visit https://keptn.sh for more information.`
+const newIncompatibleVersionMsg = `keptn version %s is available! Please note that this version is incompatible with your Keptn cluster` +
+	`version and requires to update the cluster too. Please visit https://keptn.sh for more information.`
```
Comment (christian-kreuzberger-dtx) [inline cli/utils/version/version_checker.go]: Please add a space before `version`, at the moment this statement is printed as
``Please note that this version is incompatible with your Keptn clusterversion``

In addition, please rephrase this sentence to "this version might be incompatible"

File: cli/utils/version/version_checker.go (hunk 1 above)
Comment (agrimmer) [reply cli/utils/version/version_checker.go]: Good idea! Done.

File: cli/utils/version/version_checker.go (hunk 2 above)
Comment (agrimmer) [reply cli/utils/version/version_checker.go]: Corrected the spelling issue.
Right, currently "might" is currently correct.

**Pull request merged**